tests/test_basic_load.py

```diff
@@ -34,6 +34,11 @@ def test_folder_prompts(test_folder_location):
         assert 'hello' in loaded_keys and 'test' in loaded_keys
         assert 'subfolder/nested'in loaded_keys
 
+    try:
+        print(base_prompts.nonexistentprompt)
+    except ValueError:
+        pass
```

Comment:
Using [`pytest.raises`](https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest-raises) is better:

```
with pytest.raises(ValueError):
    print(base_prompts.nonexistentprompt)
```

If you need to inspect the exception object raised you can also do:

```
with pytest.raises(ValueError) as some_error:
    print(base_prompts.nonexistentprompt)
some_error.value  # exception caught
```